packages/generators/templates/shared/.eslintrc.js.hbs

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
I think we should take a second look at this file. If this is important common configuration for all packages, we should add it to the root eslintrc file.

Reply:
I saw lots of packages define this rule, so I think we can add it to the root .eslintrc file and keep this file empty. What do you think?

Reply:
Hm. On second look, I wonder if we should just remove it and allow a package to add it if needed. Looking at the rule, it kind of seems like it should be very rarely used: https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-mocks-import.md

Reply:
If you agree, I'd recommend removing this file

Reply:
👍